OpenRA.Mods.Common/UtilityCommands/DumpSequenceSheetsCommand.cs

```diff
@@ -51,13 +51,16 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			var count = 0;
 
 			var sb = sequences.SpriteCache.SheetBuilders[SheetType.Indexed];
-			foreach (var s in sb.AllSheets)
+			if (sb.Current != null)
```

Comment:
Is this actually needed? I would have thought that `sb.AllSheets` should still work fine (returning an empty set) even when current == null.

Reply:
as you can see in the code bellow we need Current to determine wether the sheet is packed or not

Reply:
That code is inside the loop, which won't run if there are no sheets.

Reply:
I know I added it as it originally crashed. Now the command seems to run on all mods without this